exercises/practice/resistor-color/.meta/example.rs

```diff
@@ -0,0 +1,30 @@
+use int_enum::IntEnum;
+
+#[repr(u8)]
+#[derive(Clone, Copy, Debug, Eq, PartialEq, IntEnum)]
+pub enum ResistorColor {
+    Black = 0,
+    Brown = 1,
+    Red = 2,
+    Orange = 3,
+    Yellow = 4,
+    Green = 5,
+    Blue = 6,
+    Violet = 7,
+    Grey = 8,
+    White = 9,
+}
+
+pub fn color_to_value(color: ResistorColor) -> u8 {
+    color as u8
```

Comment:
```suggestion
    color.int_value()
```

This improves things in case you ever change the internal representation.